modules/3d-tiles/src/types.ts

```diff
@@ -49,3 +49,36 @@ type BoundingVolume = {
   box?: number[];
   sphere?: number[];
 };
+
+/**
+ * 3DTILES_implicit_tiling types
+ * Spec - https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_implicit_tiling#subtree-file-format
+ */
+export type SubtreeAvailability = {
+  buffers: Buffer[];
+  bufferViews: BufferView[];
+  tileAvailability: Availability;
+  contentAvailability: Availability;
+  childSubtreeAvailability: Availability;
+};
+
+export type Availability = {
+  constant?: 0 | 1;
+  bufferView?: number;
+  // Internal bitstream type
+  explicitBitstream?: ExplicitBitstream;
+};
+
+export type ExplicitBitstream = number[];
```

Comment:
Should this be booleans?  I assume you are following the spec so maybe this is better.
```suggestion
export type ExplicitBitstream = boolean[];
```

Reply:
Most compact way of storing this would probably be a Uint8Array with one byte for each boolean.

Reply:
Done